shared/ghutil/issue.go

```diff
@@ -199,6 +199,19 @@ func (gc *GithubClient) EditComment(org, repo string, commentID int64, commentBo
 	return err
 }
 
+// DeleteComment deletes comment from issue
+func (gc *GithubClient) DeleteComment(org, repo string, commentID int64) error {
```

Comment:
I think you need to add this to the Interface to make unit test work

Reply:
fixed